openscenario/openscenario_interpreter/src/syntax/story.cpp

```diff
@@ -34,6 +35,19 @@ Story::Story(const pugi::xml_node & node, Scope & scope)
   });
 }
 
+auto Story::run() -> void
+{
+  for (size_t index{0}; auto && act : elements) {
```

Comment:
This repository is compiled with C++17. Don't take advantage of C++20 features yet.
```suggestion
  std::size_t index = 0;
  for (auto && act : elements) {
```